packages/grid/_modules_/grid/components/panel/filterPanel/FilterForm.tsx

```diff
@@ -200,7 +201,7 @@ export function FilterForm(props: FilterFormProps) {
         >
           {currentColumn?.filterOperators?.map((operator) => (
             <option key={operator.value} value={operator.value}>
-              {operator.label}
+              {apiRef!.current.getLocaleText(operator.value as TranslationKeys)}
```

Comment:
I think that we should prefix these keys. Otherwise, it will be a lot harder to refactor in the future (find where the keys are used in the codebase with our search tools)

```suggestion
              {apiRef!.current.getLocaleText(`operator${capitalize(operator.value) as TranslationKeys)}`}
```

I had written https://github.com/oliviertassinari/i18n-extract in the past, I can say from experience that it gets quickly out of hand on an end-products. This script uses Babel to parse the AST and extract the use translation keys by the codebase.

Reply:
Thanks for the suggestion, I agree to prefix them will be better.